stdlib/weak.mli

```diff
@@ -112,6 +112,15 @@ val blit : 'a t -> int -> 'a t -> int -> int -> unit
     the values and give the same result as with the values themselves.
     *)
 
+(** {b Unsynchronized accesses}
+
+    Unsynchronized accesses of weak hash sets is a programming error.
```

Comment:
The alert is missing here. Any particular reason for this?

Reply:
The alert should be on the module produced by the `Make` functor which is not currently supported as far as I can see.